README.md

```diff
@@ -28,7 +30,20 @@ You can run the project using the following command. You will need to install No
 npm install
 ```
 
-1. Start the Next.js Application
+2. Setup Firebase Environment Values
```

Comment:
@arkadiuszbachorski I have added some more information about the `.env` setup here as I updated the README. Might make sense to integrate this with a local Firebase emulator setup & add some default values for this in this repo as we do it for the iOS and other apps.